yarn.lock

```diff
@@ -1421,6 +1421,15 @@ babel-plugin-jest-hoist@^24.6.0:
   dependencies:
     "@types/babel__traverse" "^7.0.6"
 
+babel-plugin-replace-ts-export-assignment@^0.0.1:
+  version "0.0.1"
+  resolved "https://registry.yarnpkg.com/babel-plugin-replace-ts-export-assignment/-/babel-plugin-replace-ts-export-assignment-0.0.1.tgz#78305d7e15fe1cbfff7bbbfdf7f0cf92f21a15d6"
+  integrity sha512-YBhWpHLAveoXQSDiiU0lk1pPDHKeBpDTcwjUTdzFOTe40kAGxcSM+XvD7Vp6SP9D79eBKNHgG5Ji3cigPjAcfw==
+  dependencies:
+    "@babel/core" "^7.5.5"
```

Comment:
`@babel/core` should be a peer dep, not a dep (if you need it).

---

Why do you need any deps at all, btw?

Reply:
Yeah that was something I debated over, and was a little unsure of, as they're needed for the types.

But looking over it you're completely right - for some reason I thought I imported one thing from `babel` that meant I need a dep instead of a dev (i.e something like AST_TREE_NODE, where it was just referencing a constant) but no I don't.

I'll make them dev deps & publish a new version :)

Reply:
I've released a new version w/o any dependencies :)